bin/sass-flatten

```diff
@@ -9,26 +9,42 @@ var write = fs.writeFileSync;
 
 var flatten = require('../');
 var meow = require('meow');
-var stdin = require('get-stdin');
-
-var cli = meow({
-  pkg: '../package.json',
-  help: [
-    'Usage',
-    '  sass-flatten <input.scss> [output.scss]',
-    'Options',
-    '  --include-path            Path to look for imported files'
-  ].join('\n')
+var getStdin = require('get-stdin');
+
+// var cli = meow({
+//   pkg: '../package.json',
+//   help: [
+//     'Usage',
+//     '  sass-flatten <input.scss> [output.scss]',
+//     'Options',
+//     '  --include-path            Path to look for imported files'
+//   ].join('\n')
+// });
+
+var cli = meow(`
+Usage
+  sass-flatten <input.scss> [output.scss]
+
+Options
+  --include-path            Path to look for imported files
+`, {
+  pkg: require('../package.json')
 });
 
 var src = cli.input[0];
 var dest = cli.input[1] || src;
+
+if (!src) {
```

Comment:
Before, this would break on `path.dirname(src)` since `src` would be undefined if not passed in.